src/lib/awsElasticSearch/index.js

```diff
@@ -277,7 +265,7 @@ const deleteIndex = async (indexName, passedClient) => {
     logger.info(`AWS OpenSearch: Successfully deleted index '${indexName}'`);
     return res;
   } catch (error) {
```

Comment:
```js
  } catch (error) {
    const alreadyExisted = error?.meta?.body?.error?.type === 'index_not_found_exception';
    if (!alreadyExisted) {
      auditLogger.error(`AWS OpenSearch Error: Unable to delete index '${indexName}': ${error.message}`);
      throw error;
    }
    return res;
  }
```
why bother checking, why only throw an error in this one case? just curious

Reply:
If the index doesn't exist there is nothing to delete. I guess the thinking was its not an error and should continue on the execution path. For the other path there is a chance the index does exist and a real error has occurred when attempting to delete it.

Reply:
gotcha, makes sense